01week/resume/index.html

```diff
@@ -2,37 +2,93 @@
 
 <html>
 <head>
-  <title> </title>
+  <title> Resume 2017 </title>
 </head>
 <body>
+<style>
+img { display: block;
+      margin: auto;}
+
+a {text-decoration: none; //}
+
+p  { font-family: 'Georgia', serif;
+     text-align: justify;
+     margin-left: 100px;
+     margin-right: 75px;}
+
+h1 { text-align: center;
+     font-family: 'Palatino', serif;
+     font-size: 4.00em;
+     margin-bottom: 15px;
+     color: darkblue}
+h2 {color: darkblue;}
+nav {text-align: center;}
+</style>
+
 <header>
-  <h1> Isela Triana </h1>
-  <nav>
-    Home
-    LinkenIn
-    <a href="mailto:anahaitriana@gmail.com">anahaitriana@gmail.com</a>
+  <h1 > Isela Triana </h1>
```

Comment:
Make sure that if you are not adding any attribute to an element avoid having spaces 
No - `<h1 >`
Yes - `<h1>`